common.sh

```diff
@@ -203,9 +213,14 @@ if env | grep -q "_LOCAL_IMAGE=" ; then
     export MIRROR_IMAGES=true
 fi
 
-if [ -n "$MIRROR_IMAGES" ]; then
-    # We're going to be using a locally modified release image
-    export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="${LOCAL_REGISTRY_DNS_NAME}:${LOCAL_REGISTRY_PORT}/localimages/local-release-image:latest"
+if [[ ! -z "${OC_MIRROR}" ]] && [[ "${OC_MIRROR}" == "true" ]]; then
+    export MIRROR_IMAGES=true
+
+    if [[ -x "/usr/local/bin/oc-mirror" ]]; then
+       # set up the channel using the most recent candidate release
+       release_candidate=`oc-mirror list releases --channel=candidate-${OPENSHIFT_RELEASE_STREAM} | tail -1`
+       export OPENSHIFT_RELEASE_TAG="${release_candidate}-x86_64"
+    fi
```

Comment:
Could also this one be removed and kept just the one define in 01_agent_requirements?

Reply:
Good catch, yes we should have that only defined on one place, will fix.